python/graphstorm/dataloading/dataset.py

```diff
@@ -156,6 +186,72 @@ def edge_feat_field(self):
         """the field of edge feature"""
         return self._edge_feat_field
 
+    def load_wg_feat(self, part_config, num_parts, ntype, rank, num_ranks):
+        """Load features from wholegraph memory
+
+        Parameters
+        ----------
+        part_config : str
+            The path of the partition configuration file.
+        num_parts : int
+            The number of partitions of the dataset
+        ntype: str
+            The type of node for which to fetch features.
+        rank: int
+            The rank of the process
+        num_ranks: int
+            The total number of ranks (world size)
+        """
+        import pylibwholegraph.torch as wgth
+        import pylibwholegraph.binding.wholememory_binding as wmb
+
+        if not self._wg_init:
+            class Options:
+                """Set Wholegraph's environment variables """
```

Comment:
Nit: Can you adjust the indent of your comment?

Reply:
Fixed.